mediacapture-streams/obtaining-local-multimedia-content/mediadevices/api-present.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
```

Comment:
After discussion with dom, we decided to follow the spec chapter layout. I did that exactly.